feature-group-definitions/declarative-shadow-dom.yml

```diff
@@ -0,0 +1,8 @@
+name: Declarative shadow DOM
+description: The `shadowrootmode` attribute on `<template>` creates a shadow DOM without the use of JavaScript. It is a declarative alternative to the `attachShadow()` method.
```

Comment:
@mfreed7 is this description accurate? The goal here is to explain the feature in one or two sentences to web developers. It doesn't need to be extremely precise, but shouldn't be misleading of course.

Reply:
For comparison, caniuse describes this more in terms of the use case:

> Proposal to allow rendering elements with shadow dom (aka web components) using server-side rendering.

Reply:
I think the description as-is is fairly accurate. And I like it better than caniuse's description, because there are other use cases also. This description describes what the feature *does*, not how it can be used.

One nit: use`creates a shadow **root**` instead of a `shadow DOM`.

Reply:
Great, thanks for the review! I'll update the wording.